src/main/steps/applicant2Sequence.ts

```diff
@@ -1,13 +1,39 @@
+import { YesOrNo } from '../app/case/definition';
+
 import { Step } from './applicant1Sequence';
-import { HAS_RELATIONSHIP_BROKEN_APPLICANT2, HOME_URL, YOU_NEED_TO_REVIEW_YOUR_APPLICATION } from './urls';
+import {
+  APPLICANT_2,
+  HAS_RELATIONSHIP_BROKEN_URL,
+  HOME_URL,
+  NOT_CONFIRMED_JOINT_APPLICATION,
+  RELATIONSHIP_NOT_BROKEN_URL,
+  YOU_NEED_TO_REVIEW_YOUR_APPLICATION,
+} from './urls';
 
-export const applicant2Sequence: Step[] = [
+const sequences: Step[] = [
   {
     url: YOU_NEED_TO_REVIEW_YOUR_APPLICATION,
-    getNextStep: () => HAS_RELATIONSHIP_BROKEN_APPLICANT2,
+    getNextStep: () => HAS_RELATIONSHIP_BROKEN_URL,
+  },
+  {
+    url: HAS_RELATIONSHIP_BROKEN_URL,
+    // TODO - to be replaced with Has your marriage irretrievably broken down page once developed
```

Comment:
Looks like this `TODO` is done? 🎉

Reply:
What is the next page for this page?

Reply:
For this "relationship broken down" page AFAIK it goes to the "enter your name" page, which I have a PR https://github.com/hmcts/nfdiv-frontend/pull/731 for that adds this to this file. The Miro board should have the full sequence.

But this comment looks like it should be removed since its saying add the broken down page here, which has been added as `HAS_RELATIONSHIP_BROKEN_APPLICANT2`.

Reply:
yhh, ill fix the comment and as its in your pr, ill leave the nextpage.